### PR TITLE
factor out default disaster to standalone file for easy access

### PR DESCRIPTION
**Pull request by juliexxia**
@ruthtalbot not sure where documentation should go for this. I documented lightly in file but I figure more extensive docs will have to live somewhere else

### CODE REVIEW

File: docs/resources.js [hunk 1]
```diff
@@ -7,11 +8,11 @@ export {getBackupScoreAssetPath, getDisaster, getScoreAssetPath};
  * @return {string} current disaster
  */
 function getDisaster() {
-  if (localStorage.getItem('disaster')) {
+  if (localSgtorage.getItem('disaster')) {
```
Comment (janakdr) [inline docs/resources.js]: Typo

File: docs/default_disaster.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export {
+  DEFAULT_DISASTER
```
Comment (janakdr) [inline docs/default_disaster.js]: Can we one-line this?

Also, could violate convention and put at end of file so line user has to edit is closer to top?

Comment (juliexxia) [reply docs/default_disaster.js]: I thought about that too. Went for it!

**Comment by janakdr** [comment]
#406

**Pull request merged**